candidates/tests/test_constituency_lock.py

```diff
@@ -13,8 +13,9 @@ class TestConstituencyLockAndUnlock(TestUserMixin, UK2015ExamplesMixin, WebTest)
 
     def setUp(self):
         super(TestConstituencyLockAndUnlock, self).setUp()
-        self.camberwell_post_extra.candidates_locked = True
-        self.camberwell_post_extra.save()
+        election_extra = self.camberwell_post_extra.postextraelection_set.all()[0]
```

Comment:
Another note about using `postextraelection` or similar instead of `election_extra`, and preferring `.get()` to `.all()[0]` since we expect exactly one.